qemu/tests/cfg/openflow_acl_test.cfg

```diff
@@ -42,7 +42,6 @@
             access_cmd = ssh
             target_port = 22
             host_username = root
-            host_password = redhat
```

Comment:
I think it's ok to keep this params, user would update it according to their env

Reply:
Users may probably forget or don't know they need to modify this parameter before they do the test. Then they'll get test fail. I don't agree this is a good way to users.